src/Plug.vala

```diff
@@ -35,53 +34,48 @@ public class ApplicationsPlug : Switchboard.Plug {
         settings.set ("applications/defaults", DEFAULTS);
         settings.set ("applications/startup", STARTUP);
         settings.set ("applications/permissions", PERMISSIONS);
-        Object (category: Category.PERSONAL,
-                code_name: "io.elementary.switchboard.applications",
-                display_name: _("Applications"),
-                description: _("Manage default apps, startup apps, and app permissions"),
-                icon: "preferences-desktop-applications",
-                supported_settings: settings);
+
+        Object (
+            category: Category.PERSONAL,
+            code_name: "io.elementary.switchboard.applications",
+            description: _("Manage default apps, startup apps, and app permissions"),
+            display_name: _("Applications"),
+            icon: "preferences-desktop-applications",
+            supported_settings: settings
+        );
     }
 
     public override Gtk.Widget get_widget () {
-        if (grid != null) {
-            return grid;
+        if (grid == null) {
+            stack = new Gtk.Stack () {
+                expand = true
+            };
+            stack.add_titled (new Defaults.Plug (), DEFAULTS, _("Defaults"));
+            stack.add_titled (new Startup.Plug (), STARTUP, _("Startup"));
+            stack.add_titled (new Permissions.Plug (), PERMISSIONS, _("Permissions"));
+
+            var stack_switcher = new Gtk.StackSwitcher () {
+                halign = Gtk.Align.CENTER,
+                homogeneous = true,
+                margin_top = 12,
+                stack = stack
+            };
+
+            grid = new Gtk.Grid () {
+                row_spacing = 24
```

Comment:
Was the tiny margin_top deliberately omitted?

Reply:
Yes, I'm not sure why that was there in the first place :)